lib/run/summary.js

```diff
@@ -2,7 +2,11 @@ var _ = require('lodash'),
     SerialiseError = require('serialised-error'),
 
     proxyPropertyList,
-    RunSummary;
+    RunSummary,
+    exportReference = {
```

Comment:
What is this? Should not global and environment be treated using same code base?